rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -207,7 +202,7 @@ public void receiveHeaders(BtcBlock[] headers) throws IOException {
             } catch (Exception e) {
                 // If we tray to add an orphan header bitcoinj throws an exception
                 // This catches that case and any other exception that may be thrown
-                logger.warn("Exception adding btc header", e);
+                logger.warn("Exception adding btc header {}", headers[i].getHash(), e);
```

Comment:
Not sure if we want to add this to the logger to be honest, this might be a bit expensive and if they send us a bunch of orphan blocks we would be doing it by each of those blocks...

Reply:
but if we don't do it we won't know what header was the one that failed